src/components/molecules/user-result/user-result.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Avatar from 'components/atoms/avatar';
+
+import {
+  UserResultContainer,
+  AvatarContainer,
+  DetailsContainer,
+  StatsContainer,
+  Row,
+  NameDisplay,
+  UsernameDisplay,
+} from './user-result.styles';
+
+const UserResult = ({
+  avatarUrl,
+  email,
+  followersCount,
+  followingCount,
+  location,
+  login,
+  name,
+  starredRepositoriesCount,
+  url,
+}) => (
+  <UserResultContainer href={url} target="_blank">
```

Comment:
See comment on OrganizationResultContainer for improvements that should be made.